src/front/cobalt.js

```diff
@@ -670,6 +671,7 @@ window.onload = () => {
     if (isIOS) {
         document.addEventListener('touchstart', () => {}, true);
     }
+    eid("url-input-area").focus();
```

Comment:
input area is already focused when you press ctrl/cmd (as part of ctrl+v shortcut). this is unnecessary.

Reply:
huh, never noted that